packages/generator/test/generated/update.ts

```diff
@@ -1,15 +1,21 @@
 // this file will replace all the expected.js files with their _actual equivalents
 
 import { readFileSync, writeFileSync } from 'fs'
+import { fileURLToPath } from 'url'
+import { dirname } from 'path'
 import glob from 'tiny-glob/sync.js'
 
 console.log(`
 updating generated files ...`
 )
 
-glob('**/*.actual.*', { cwd: __dirname }).forEach((file) => {
-	const actual = readFileSync(`${__dirname}/${file}`, 'utf-8')
-	writeFileSync(`${__dirname}/${file.replace('.actual.', '.expected.')}`, actual)
+const cwd = typeof __dirname !== 'undefined'
+  ? __dirname
+  : dirname(fileURLToPath(import.meta.url))
+
+glob('**/*.actual.*', { cwd }).forEach((file) => {
+	const actual = readFileSync(`${cwd}/${file}`, 'utf-8')
+	writeFileSync(`${cwd}/${file.replace('.actual.', '.expected.')}`, actual)
```

Comment:
Had an issue where __dirname was not defined hence this fix.

Reply:
Ah I forgot to fix this file before I pushed it. I reverted it to the state from before yesterday